stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -1201,13 +1201,14 @@ factorize(A::Transpose) = transpose(factorize(parent(A)))
 ## Moore-Penrose pseudoinverse
 
 """
-    pinv(M[, rtol::Real])
+    pinv(M, atol::Real, rtol::Real)
```

Comment:
Should this be keyword args?

Reply:
Sorry for the delay @andyferris. I resolved this now. I have some issues regarding the test/pinv tests. I will try to resolve them soon.